hub/core/chunk/chunk_compressed_chunk.py

```diff
@@ -211,6 +220,12 @@ def update_sample_img_compression(self, local_index: int, new_sample: InputSampl
         self.update_in_meta_and_headers(local_index, None, shape)
 
     def process_sample_img_compr(self, sample):
+        if sample is None:
+            if self.tensor_meta.max_shape:
+                shape = (0,) * len(self.tensor_meta.max_shape)
+            else:
+                shape = (0, 0, 0)
```

Comment:
Why 3?

Reply:
if there's no non None sample before this then we have no way of knowing what the number of dimensions is going to be.
But seeing that this uses image chunk compression, isn't it always gonna be 3d?

Reply:
Can be 2d too right?